lib/DDG/Spice/Pokemon/Data.pm

```diff
@@ -0,0 +1,31 @@
+package DDG::Spice::Pokemon::Data;
+# ABSTRACT: Returns pokemon data from the pokeapi API
+
+use DDG::Spice;
+
+spice is_cached => 1;
+spice proxy_cache_valid => "200 30d";
+
+name "Pokemon";
+source "Pokéapi";
+icon_url "/i/www.pokeapi.co.ico";
+description "Fetches details of a given Pokemon";
+primary_example_queries "pokemon pikachu", "bulbasaur pokemon";
+category "entertainment";
+topics "entertainment", "geek";
+code_url "https://github.com/duckduckgo/zeroclickinfo-spice/blob/master/lib/DDG/Spice/Pokemon.pm";
+attribution github => ["AkA84"];
+
+# Triggers
+triggers startend => "pokemon";
+
+spice to => 'http://pokeapi.co/api/v1/pokemon/$1';
```

Comment:
Please add a `/` at the end of `$1` because http://pokeapi.co/api/v1/pokemon/pikachu redirects to it. We can't handle redirects in production machines unfortunately.